### ISSUE TITLE
Failing to build on ARM

### ISSUE THREAD

**Issue by poperigby** [bluescan/tacentview#191]
When testing the compilation of Tacent View on ARM in the NixOS CI, the following error is being produce:

```
[8/33] Building CXX object UnitTests/CMakeFiles/UnitTests.dir/Src/UnitTests.cpp.o
FAILED: UnitTests/CMakeFiles/UnitTests.dir/Src/UnitTests.cpp.o
/nix/store/ay4xxbjafns12mn2ilhz4y7d3m3ziwqg-gcc-wrapper-13.3.0/bin/g++ -DCONFIG_RELEASE -DPLATFORM_LINUX -I/build/source/UnitTests/Inc -I/build/source/Modules/Foundation/Inc -I/build/source/Modules/Math/Inc -I/bu>
In file included from /build/source/UnitTests/Src/UnitTests.cpp:20:
/build/source/UnitTests/Src/UnitTests.cpp: In function 'int main(int, char**)':
/build/source/UnitTests/Src/UnitTests.cpp:149:15: error: 'Rule' is not a member of 'tUnitTest'
  149 |         tTest(Rule);
      |               ^~~~
/build/source/UnitTests/Src/UnitTests.h:48:146: note: in definition of macro 'tTest'
   48 | #define tTest(name)  { tUnitTest::rPrintf("\nTesting " #name "\n"); tUnitTest::UnitRequirementNumber = tUnitTest::UnitGoalNumber = 0; tUnitTest::name(); }
      |                                                                                                                                                  ^~~~
```

Any ideas?

**Comment by bluescan** [comment]
Yeah. Unfortunately ARM isn't supported. Even though I likely fixed the above in 701817ceeec6ee76420a57da9529943480d0047e, it won't matter since the entire Tacent Image module is only available for x64, not ARM... at least for the forseeable future.

**Comment by poperigby** [comment]
Understandable. Thank you!
